Spigot-API-Patches/0167-Reset-players-remainingAir-on-respawn.patch

```diff
@@ -0,0 +1,21 @@
+From 7ac07ac07ac07ac07ac07ac07ac07ac07ac07ac0 Mon Sep 17 00:00:00 2001
+From: GreenMeanie <GreenMeanieMC@gmail.com>
+Date: Sat, 20 Oct 2018 21:20:48 -0400
+Subject: [PATCH] Reset players remainingAir on respawn
+
+
+diff --git a/src/main/java/org/bukkit/event/player/PlayerRespawnEvent.java b/src/main/java/org/bukkit/event/player/PlayerRespawnEvent.java
+index 7ac07ac07ac0..7ac07ac07ac0 100644
+--- a/src/main/java/org/bukkit/event/player/PlayerRespawnEvent.java
++++ b/src/main/java/org/bukkit/event/player/PlayerRespawnEvent.java
+@@ -17,6 +17,7 @@ public class PlayerRespawnEvent extends PlayerEvent {
+         super(respawnPlayer);
+         this.respawnLocation = respawnLocation;
+         this.isBedSpawn = isBedSpawn;
++        respawnPlayer.setRemainingAir(300);
```

Comment:
Please add // Paper

Reply:
I scrapped this. I will update this when I finish writing